servicemesh-install/topics/tutorial-jaeger-tracing.adoc

```diff
@@ -0,0 +1,44 @@
+[[distributed_tracing_tutorial]]
+= Distributed Tracing Tutorial
+////
+TODO
+Need a short overview for distributed tracing and Jaeger.
+////
+This tutorial uses {ProductShortName} and the `bookinfo` tutorial to demonstrate how you can perform a trace using the Jaeger component of {ProductName}.
+
+Prerequisites:
+
+* OpenShift Container Catalog 3.10 or higher installed.
+* {ProductName} {ProductVersion} installed.
+* `Bookinfo` demonstration application installed.  
+
+
+== Generating Traces and Analyzing Trace Data
+
+. Once you've deployed the Bookinfo application, generate some activity by accessing http://$GATEWAY_URL/productpage and refreshing the page a few times.
+.  A route to access the Jaeger dashboard should already exist. Query OpenShift for details of the route:
++
+```
+  $ oc get route jaeger-query -n istio-system
+  NAME           HOST/PORT                                              PATH      SERVICES       PORT           TERMINATION   WILDCARD
+  jaeger-query   jaeger-query-istio-system.127.0.0.1.nip.io             jaeger-query   jaeger-query   edge          None
+
+```
++
+. Launch a browser and navigate to https://jaeger-query-istio-system.127.0.0.1.nip.io.
```

Comment:
This only works in 'oc cluster up' environments, it would be better to have something like

export JAEGER_URL=$(oc get route -n istio-system jaeger-query -o jsonpath='{.spec.host}')
navigate to https://${JAEGER_URL}